src/axolotl/cli/main.py

```diff
@@ -48,6 +51,15 @@ def train(config: str, accelerate: bool, **kwargs):
     """Train or fine-tune a model."""
     kwargs = {k: v for k, v in kwargs.items() if v is not None}
 
+    # enable expandable segments for cuda allocation to improve VRAM usage
+    torch_version = torch.__version__.split(".")
+    torch_major, torch_minor = int(torch_version[0]), int(torch_version[1])
+    if torch_major == 2 and torch_minor >= 2:
+        if os.getenv("PYTORCH_CUDA_ALLOC_CONF") is None:
+            os.environ[
+                "PYTORCH_CUDA_ALLOC_CONF"
+            ] = "expandable_segments:True,roundup_power2_divisions:16"
```

Comment:
I'm just reading about this: `roundup_power2_divisions` config. Is there a reason we're setting 16? I saw that the default was 512.

Reply:
The default is 1 from what I've seen. 

The doc you were reading says :  the sizes are rounded up in multiple of blocks size of 512

Which is the block size, not the divisions